guidelines/releases/release-process.md

```diff
@@ -7,102 +7,183 @@ This document describes how to create a Kyma release using Prow.
+
+    2. check all `yaml` files for references in the form of:  
+        `image: eu.gcr.io/kyma-project/develop/{{IMAGE_NAME}}:{{SOME_SHA}}`
+        and change them to:  
+        `image: eu.gcr.io/kyma-project/{{IMAGE_NAME}}:{{RELEASE_NAME}}`
+
+        known files to change:
+        - `installation/resources/installer-local.yaml`
+        - `installation/resources/watch-pods.yaml`
+        - `resources/application-connector-ingress/templates/upgrade-job.yaml`
+
+        Don't change the following reference:
+        - `installation/resources/installer.yaml`:  
 
-![](../../assets/release-PR.png)
-This triggers all jobs for components.
-Update your PR with the version and the directory of components used in `values.yaml` files.
+            ```yaml
+            apiVersion: extensions/v1beta1
+            kind: Deployment
+            metadata:
+              name: kyma-installer
+              namespace: kyma-installer
+              labels:
+                kyma-project.io/installation: ""
+            spec:
+              template:
+                metadata:
+                  labels:
+                    name: kyma-installer
+                spec:
+                  serviceAccountName: kyma-installer
+                  containers:
+                  - name: kyma-installer-container
+                    image: eu.gcr.io/kyma-project/develop/installer:55bc6038
+                    imagePullPolicy: IfNotPresent
+             ```
 
-Change these values in the files:
+3. If any job fails, retrigger it by adding the following comment to the PR:
 
-```
-dir: develop/
-version: {current_version}
-```
-Replace them with:
-```
-dir:
-version: {rel_version}
-```    
+    ```;
+    /test {job_name}
+    ```
 
-Every component image is published with a version defined in the `RELEASE_VERSION` file stored in the `test-infra` repository on the given release branch.
+    > **IMPORTANT:** Never use `/test all` as it might run tests that you do not want to execute!
 
-5. If any job fails, retrigger it by adding the following comment to the PR:
-```
-/test {job_name}
-```
+4. Wait until all jobs for components and tools finish.
+5. Execute remaining tests
+    ![JobDependencies](../../assets/kyma-rel-jobs.svg)
+    1. Run `kyma-integration` by adding the  `/test pre-{release_number}-kyma-integration`  comment to the PR.
+        > **NOTE:** You don't have to wait until the `pre-{release_number}-kyma-integration` job finishes.
 
-6. Wait until all jobs for components and tools finish. You must run the
-`kyma-integration`, `kyma-gke-integration`, `kyma-gke-upgrade`, `kyma-artifacts`, and `kyma-installer` jobs manually because there are dependencies between them. See the diagram for details:
+    2. Run `kyma-installer`
+    ```/test pre-{release_number}-kyma-installer``` and wait until it finishes
+    3. Run `kyma-artifacts`
```

Comment:
As above, use either 

```
Run `kyma-artifacts`.  

```
OR
```
Run `/test pre-{release_number}-kyma-artifacts` and wait until it finishes. 
```
and remove the next line.

Reply:
option 2